ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SubjectComparisonVerifier.kt

```diff
@@ -99,16 +103,16 @@ class SubjectComparisonVerifier(private val request: Node? = null, private val r
     @Suppress("NestedBlockDepth" /* Simple `let` nesting */)
     fun verifySubjectsMatchAuthnRequest() {
 
-        val requestSubject = request?.children(SUBJECT)?.firstOrNull() ?: return
+        val requestSubject = samlRequest?.dom?.children(SUBJECT)?.firstOrNull() ?: return
```

Comment:
I chose to just pull out the dom here instead of using the openSaml objects because we haven't yet figured out if we want to majorly use openSaml yet.

Reply:
Then why did we change the method to take an opensaml object in the first place? 😂

Reply:
To pass around the dom :trollface: